lib/internal/bootstrap/node.js

```diff
@@ -95,6 +95,11 @@ if (isMainThread) {
   process.dlopen = rawMethods.dlopen;
   process.uptime = rawMethods.uptime;
 
+  Object.defineProperty(process, 'exitWithException', {
+    enumerable: true,
+    value: rawMethods.exitWithException
```

Comment:
```suggestion
    configurable: false,
    writable: false,
    value: rawMethods.exitWithException
```

Or otherwise, do not call this internally but use the one from the internalBinding instead to avoid potential monkey-patching?

Reply:
@joyeecheung `configurable` is `false` by default, according to MDN.